### ISSUE TITLE
[perf] Change «Reopen in X container» animation

### ISSUE THREAD

**Issue by andsofine** [mbnuqw/sidebery#1743]
### Description
I think it is worth avoiding shifting a bunch of tabs which leads to performance degradation in a given scenario.

### What it looks like now:
1. Click "Reopen in X container".
2. A new tab opens in the desired container.
3. All the tabs at the bottom move down because another tab has appeared.
4. Closes the old tab in the old container.
5. All the tabs at the bottom move again but upwards because a tab closed.

### How I propose this should look like:
1. The old tab is closed but its space remains unoccupied.
2. On the unoccupied place opens a new tab in the desired container. 

PS. There is also an option to make that there was even less animation and the tab simply changed the color of the container indicator. But I don't think this is a good solution, as I guess that such a change should be accompanied by something more noticeable.

**Comment by llc0930** [comment]
I guess it's not just an animation... but something that actually happens in your browser...

**Comment by andsofine** [comment]
> I guess it's not just an animation... but something that actually happens in your browser...

Yes maybe it's not the root of the problem but I think it's better to avoid unnecessary animation of shifting all tabs under the reopened one.

(Very sorry for the late reply)
